source/simulator/parameters.cc

```diff
@@ -1564,10 +1564,18 @@ namespace aspect
         nonlinear_solver = NonlinearSolver::no_Advection_no_Stokes;
       else if (solver_scheme == "no Advection, single Stokes")
         nonlinear_solver = NonlinearSolver::no_Advection_single_Stokes;
-      // deprecated: remove "first timestep only, single Stokes" eventually
-      else if (solver_scheme == "no Advection, single Stokes first timestep only" ||
-               solver_scheme == "first timestep only, single Stokes")
+      else if (solver_scheme == "no Advection, single Stokes first timestep only")
         nonlinear_solver = NonlinearSolver::no_Advection_single_Stokes_first_timestep_only;
+      // deprecated: use "no Advection, single Stokes first timestep only" instead
+      else if (solver_scheme == "first timestep only, single Stokes")
+        {
+          if (Utilities::MPI::this_mpi_process(mpi_communicator) == 0)
+            {
+              std::cout << "Warning: You are using the deprecated spelling <first timestep only, single Stokes> " << std::endl
```

Comment:
I would call it "name" or "parameter name" or "naming scheme". It is not really spelling.